src/App.tsx

```diff
@@ -2,8 +2,21 @@ import React from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
+import { useState } from 'react';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const handleChange: React.ChangeEventHandler<HTMLInputElement> = event => {
+    setQuery(event.target.value);
+  };
+
+  const visibleMovies = moviesFromServer.filter(
+    movie =>
+      movie.title.toLowerCase().includes(query.trim().toLowerCase()) ||
```

Comment:
`query.trim().toLowerCase()` I'd prefer creating a separate variable for it, to avoid repeating